src/app/templates/base.html

```diff
@@ -7,3 +7,4 @@
         {% block content %}{% endblock %}
     </div>
```

Comment:
noticed this random </div> but i guess it wont break if you leave it alone.

Reply:
That was already in it, did not touch it and left it alone

Reply:
Div here is to ensure that everyone has the outer container, shouldn't cause issues